serverstats/serverstats.py

```diff
@@ -304,7 +304,6 @@ def _bitsize(num):
             ).format(
                 owner_mention=bold(str(owner.mention)),
                 owner=bold(str(owner)),
-                region=f"**{vc_regions.get(str(guild.region)) or str(guild.region)}**",
```

Comment:
Removing *only* this line is going to throw KeyErrors. You will also need to remove the `Region: {region}\n` in the actual string.

Reply:
Removed.